docstore/doc.go

```diff
@@ -281,6 +281,24 @@
 //
 // OpenCensus Integration
 //
-// TODO(jba): implement and document
-//
+// OpenCensus supports tracing and metric collection for multiple languages and
+// backend providers. See https://opencensus.io.
+//
+// This API collects OpenCensus traces and metrics for the following methods:
+//  - ActionList.Do
+//  - Query.Delete
+//  - Query.Get (for the first query only; drivers may make additional calls while iterating over results)
```

Comment:
This will be true for both Query.Update and Query.Delete (depending on the driver). However, I think we might want to add trace span to the entire call as the first query may only be a small portion of it. Considering we are still making changes to this API, a TODO or a separate issue will do as well.

Reply:
I don't think so... The trace is created in `Query.Update` and `Query.Delete`, so even if the driver issues multiple RPCs, the entire time is captured.

`Query.Do` is different because we explicitly return an iterator to the user, so the context in that function is just whatever time the driver takes to create the iterator. I'm pretty sure that for all drivers, that corresponds to getting the first "page" of results, or all results.

In order to get the entire iteration for `Query.Do`, we'd have to create the `ctx` in `Do`, and save it until the user calls `Stop` on the iterator. I don't think that's an improvement over what I have here, since it includes arbitrary processing time from the caller in addition to whatever time the driver/portable type take. (E.g., the user could `time.Sleep(1*time.Second)` between each call to `iter.Next`).

Reply:
You're right, they are different.

Reply:
> I'm pretty sure that for all drivers, that corresponds to getting the first "page" of results, or all results.

That is true right now, but a driver could also choose to return the iterator immediately and do the RPC on the first call to `Next`. The `driver.RunGetQuery` doc should be updated to say that drivers shouldn't do that.

Reply:
Done.